### PR TITLE
New feature issue template for GitHub

### PR DESCRIPTION
**Pull request by jrbarnes9**
### What is the context of this PR?
- Updated the 'feature request' github issues template
- Remove empty 'Research on this...' sections from all components and patterns
- Changed 'Design System Forum' sections to 'Help improve this ...' with new content to include instructions for sharing UR findings

### How to review
- Check issues template
- Check bottom of all DS pages for new content

### CODE REVIEW

**Comment by jrbarnes9** [comment]
[New feature GitHub template for contributors](https://trello.com/c/5NPoexgY/1195-new-feature-github-template-for-contributors)

**Comment by rmccar** [comment]
[New feature GitHub template for contributors](https://trello.com/c/5NPoexgY/1195-new-feature-github-template-for-contributors)

**Review by LauraC119 — COMMENTED** [review]
Added a content doc to the Trello card with content changes to the template.
Checked new 'Help to improve this...' section in a few places and looks good.

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 1]
```diff
@@ -1,7 +1,37 @@
...
 assignees: ''
 ---
+
+<!--
+
+(!) Before you fill in this proposal template
+
+- If you are suggesting a change to something that already exists in the ONS Design System, please propose it by commenting on the GitHub discussion (https://github.com/ONSdigital/design-system/discussions) for that component, pattern or style. You can get to the discussion using the direct link at the bottom of each page in the Design System.
+
+- If you are proposing something new, check the Design System backlog (https://trello.com/b/dsgMhRlq/ons-design-system) first, to make sure something similar hasn't already been proposed. Please add any examples of evidence of your own, if it has already been proposed.
```
[the remark this answers is not in the document]
Comment (jrbarnes9) [reply .github/ISSUE_TEMPLATE/feature_request.md:15]: Yeah we're discussing creating a public DS community backlog in GitHub now

**Review by rmccar — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/feature_request.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (jrbarnes9) [reply .github/ISSUE_TEMPLATE/feature_request.md]: Until we're ready for that, I've added Jo's email as a contact to request access

**Review by LauraC119 — COMMENTED** [review]
Few changes to the following paragraphs please:

If you are suggesting a change to a component, pattern or style that is already in the ONS Design System, please comment on its GitHub discussion (https://github.com/ONSdigital/design-system/discussions). You can get to the discussion using the direct link at the bottom of each page in the Design System.

If you are proposing something new, check the Design System backlog (https://trello.com/b/dsgMhRlq/ons-design-system). There may already be a proposal for something similar. You can add examples of your own evidence to an existing proposal. To request access to the backlog, email jo.van.der.plank@ons.gov.uk

Give a brief description of the new feature you want to propose for the Design System. This could be a component, pattern or style, or new documentation for an existing feature.

**Review by rmccar — APPROVED** [review]

**Review by LauraC119 — APPROVED** [review]

**Pull request merged**